src/seeds/activity.js

```diff
@@ -1 +1,64 @@
-// seeds here
+import mongoose from 'mongoose';
+
+export default [
+  {
+    _id: new mongoose.Types.ObjectId('6466ggl7fc13ae78027631cc'),
+    name: 'Hayden',
+    description: 'volley',
```

Comment:
The 'name' field for 'activity' represents the name of an activity, not a person's name, and the 'description' field describes what the activity is like.